demisto_sdk/validation/file_validator.py

```diff
@@ -572,8 +573,9 @@ def is_valid_structure(self):
                 self.validate_all_files()
         else:
             if self.file_path:
-                self.branch_name = self.get_current_working_branch()
-                self.validate_committed_files()
+                structure_validator = StructureValidator(self.file_path, predefined_scheme=find_type(self.file_path))
```

Comment:
should also validate the against the specific validator class:
IntegrationValidator, ScriptValidator, etc...